pyth_observer/event.py

```diff
@@ -143,3 +144,27 @@ async def send(self):
                         logger.error(
                             f"Failed to send Telegram message: {response_text}"
                         )
+
+
+class ZendutyEvent(Event):
+    def __init__(self, check: Check, context: Context):
+        self.check = check
+        self.context = context
+
+    async def send(self):
+        event_details = self.check.error_message()
+        summary = ""
+        for key, value in event_details.items():
+            summary += f"{key}: {value}\n"
+
+        alert_identifier = (
+            f"{self.check.__class__.__name__}-{self.check.state().symbol}"
+        )
+        state = self.check.state()
+        if isinstance(state, PublisherState):
+            alert_identifier += f"-{state.publisher_name}"
```

Comment:
I have a worry that as we don't update publishers config regularly we sometimes don't have the names, so maybe the publisher key be better here. (we can change the alert message to use name if there it is, otherwise key)

Reply:
The name seems to include the key, here's an example of an alert where the name is not there but the field has the key anyway.
![image](https://github.com/pyth-network/pyth-observer/assets/30928485/3d210f6f-5316-4bc4-889a-00cd3023047b)

Reply:
ahaa my bad then. i forgot.